cmd/influxd/run/server_test.go

```diff
@@ -891,7 +891,7 @@ func TestServer_Query_Tags(t *testing.T) {
 		&Query{
 			name:    "tag without field should return error",
 			command: `SELECT host FROM db0.rp0.cpu`,
-			exp:     `{"results":[{"error":"select statement must include at least one field"}]}`,
+			exp:     `{"results":[{}]}`,
```

Comment:
This is an odd test because they're selecting a tag without other information from a measurement. I think what you mean to test here is if they select a non-existent field, which would return an empty result set.

For a query where they select a tag that actually exists, it's almost like they should get the list of hosts back.